domain/modelmigration/state/state_test.go

```diff
@@ -59,3 +61,43 @@ func (s *migrationSuite) TestGetControllerUUID(c *gc.C) {
 	c.Check(err, jc.ErrorIsNil)
 	c.Check(controllerId, gc.Equals, s.controllerUUID.String())
 }
+
+// TestGetAllInstanceIDs is asserting the happy path of getting all instance
+// IDs for the model.
+func (s *migrationSuite) TestGetAllInstanceIDs(c *gc.C) {
```

Comment:
Can you add one test case of get all instance ids that has no rows in the database to prove that no errors occur? Covers off on the no rows comment above.

Reply:
Sure! done in c71afad04b